packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -33,8 +33,11 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
       selectedKey: 'defaultSelectedKey'
     });
 
+    const validDefaultSelectedKey: boolean = !!props.options && props.options.some(option => option.key === props.defaultSelectedKey);
```

Comment:
nit: maybe have this as a private member instead of instantiating a const in the constructor. Just a thought 🙄 and not a blocker.

Reply:
Nothing wrong... just a thought 😄 ... and I do have to review it)))